### ISSUE TITLE
[Feature Request] Ability to change Push To Talk/PTT shortcut

### ISSUE THREAD

**Issue by BloodyIron** [jitsi/jitsi-meet#3464]
Hi Folks,

My understanding is that Riot.im uses jitsi for voice stuff, and they've pointed me to here for my request ( https://github.com/vector-im/riot-web/issues/5993 )

Now, Ctrl + T is the push to talk key as it is now, however, that doesn't work for everyone.

Any chance we could get the functionality to change the PTT/Push To Talk key combination? And perhaps in a way that Riot.im can nicely call too?

With many people using Jitsi for voice comms, being able to change the keyboard shortcut is a common expectation in VoIP applications. It's literally in every other popular VoIP implementation out there. So, it'd be swell if we could get it in Jitsi too.

Please? :)

**Comment by anoadragon453** [comment]
Fwiw pretty sure 'Space' is the default push-to-talk key, not Ctrl+T, but yes +1 for being able to change it!

**Comment by saghul** [comment]
Indeed it’s space.

As for the feature request, our current implementation of keyboard shortcuts is not really clean so making it configurable is not easy at the moment.

I personally like the idea though!

**Comment by anoadragon453** [comment]
I'm looking into potentially adding this as a feature to Riot and making it configurable from there. Would still be nice to see it in native Jitsi though :)

If keyboard shortcuts can be configured on the fly then it should be too bad, right? Otherwise yeah, might be kind of hard.

**Comment by BloodyIron** [comment]
I know for sure that anyone wanting to use riot (which uses jitsi for VoIP) will want to be able to set their own push to talk key binding. So, that seems like something that should be in jitsi first perhaps?

**Comment by anoadragon453** [comment]
Sure, although I think it would be cool if we could make push-to-talk work even when Riot is minimized, which will require a little work on our side as well.

**Comment by BloodyIron** [comment]
Riot having PTT while minimised is also something I think that people won't want to compromise on! :D So many other VoIP apps already do that (Mumble/Teamspeak/Ventrilo/Discord/Skype/etc).

**Comment by anoadragon453** [comment]
I am working on it!

**Comment by skylord123** [comment]
This would be awesome. This is one of the features I am dying to have (especially working while minimized).

**Comment by anoadragon453** [comment]
This is still a useful feature for Jitsi imo. Will post an update from the Riot side on https://github.com/vector-im/riot-web/issues/5993 in a moment.

**Comment by anoadragon453** [comment]
Aww, stalebot :(

**Comment by aaronkvanmeerten** [comment]
Stablebot gets overzealous sometimes.

**Comment by Mikaela** [comment]
A bit offtopic, but are the shortcut keys such as space for PTT documented anywhere? I was coming here to request it.

**Comment by damencho** [comment]
Just type '?' in a meeting.

**Comment by BloodyIron** [comment]
But we neeeeeeeeeeeeeed it! D:

**Comment by Mikaela** [comment]
The previous time I was in need of PTT, one of my issues was chat box opening by itself or me forgetting it open and it ate the space so I sometimes spoke without anyone hearing me.

I think customizability would be the best option and I would use right Ctrl which I am used to using with Mumble and Discord as it doesn't overlap with anything else and shouldn't be eaten as long as I had Jitsi Meet focused.

**Comment by Mikaela** [comment]
Has this been fixed? This is currently my boggest problem with Jitsi Meet, especially if I drop links or something else into the chat.

**Comment by Mikaela** [comment]
@damencho @aaronkvanmeerten @saghul reading your [stale.yml](https://github.com/jitsi/jitsi-meet/blob/master/.github/stale.yml) could you label this issue as "confirmed" so there won't be need of asking about this monthly? I am assuming that you plan to implement this sometime in the future as otherwise you would have just closed/locked this and requested everyone to stop commenting.

I commented on phone 11 hours ago and this time looked at this more closely after seeing the email notification.

**Comment by sr-verde** [comment]
Is there any progress at this issue? It would be so nice in the current situation.

**Comment by saghul** [comment]
No, and it’s unlikely to happen specially now that we are all hands on deck keeping the service working for everyone.

**Comment by sr-verde** [comment]
Pity! Nevertheless, good work and wish you success!

**Comment by BloodyIron** [comment]
:/

**Comment by mcd1992** [comment]
:+1: bump before stalebot creeps in

**Comment by dXNlcm5hbWU** [comment]
+1

**Comment by Eddudos** [comment]
My honest skilll issue workaround can be found here:

https://github.com/element-hq/element-web/issues/5993#issuecomment-3622339197
